osx/setup.sh

```diff
@@ -363,8 +364,16 @@ fi
 
 # Remove the .dmg if it exists.
 test -e "$DMG_PATH" && rm "$DMG_PATH"
+
+mkdir "$RELEASE_PATH/More files"
```

Comment:
@amodia Check if the "More files" directory already exists before creating, or else this will throw an exception.